src/app/serverViewState.test.js

```diff
@@ -428,6 +428,27 @@ describe('app/serverViewState', () => {
             expect(result.validatedURL).toBe('https://server.com/');
         });
 
+        it('should not update the URL if the user is typing https://', async () => {
```

Comment:
Can we also include negative test cases like `abchttp` and verify that `https://` is appended?

Reply:
Yes I will add this.